src/drizzle/meta/0007_snapshot.json

```diff
@@ -3597,4 +3598,4 @@
     "schemas": {},
     "tables": {}
   }
-}
\ No newline at end of file
+}
```

Comment:
Should this really change?

Reply:
Yes I think it was missing some values, since it generated things that were already included in the previous migration.